src/components/home/about/TextSection.jsx

```diff
@@ -0,0 +1,26 @@
+// Material UI Imports
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import { useTheme } from '@mui/material/styles';
+
+const TextSection = () => {
+  const theme = useTheme();
+  const isSmallScreen = useMediaQuery(theme.breakpoints.up('sm'));
+
+  return (
+    <Stack spacing={3} justifyContent="center" maxWidth={{ xs: '100%', lg: '33%' }}>
+      <Typography variant="h3" component="h2" textAlign={'center'}>
+        {/* Who is CODE PDX? */}
```

Comment:
sort of partial to this text. When I look at the page on my phone the font size seems fine. I would prefer this just gets reverted and Font size can be left alone. If it gets made smaller it doesn't stand out enough from the other text and as you said.. its the same size as the rest.

Reply:
reverted